authors/authors.json

```diff
@@ -8,6 +8,14 @@
         "sympy_commits": 2636
     },
     {
+        "email": "mattpap@gmail.com",
+        "github_id": "mattpap",
+        "institution": "Continuum Analytics",
```

Comment:
@certik, I'm not sure if it matters, but in formal documents we use `Continuum Analytics, Inc.`.